src/components/compatibility/CompatibilityScoreGauge.tsx

```
@@ -0,0 +1,111 @@
+    <div className="rounded-2xl bg-white/[0.05] border border-white/10 p-6">
+      <div className="flex flex-col items-center gap-4">
+        {/* Names */}
+        {person1Name && person2Name && (
+          <p className="text-white/50 text-sm">
+            {person1Name} & {person2Name}
+          </p>
+        )}
+
+        {/* Circular gauge */}
+        <div className="relative w-36 h-36">
+          <svg className="w-full h-full -rotate-90" viewBox="0 0 120 120">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)